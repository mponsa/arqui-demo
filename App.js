import { StatusBar } from 'expo-status-bar';
import React from 'react';
import { StyleSheet, Text, View, Image, TouchableOpacity} from 'react-native';
import { grey, black } from 'color-name';

export default function App() {

  const pressFunction = () => {
    alert("Hola mundo!")
  }


  return (
    <View style={styles.container}>
        <View style={styles.headerContainer}>
          <Image style={styles.image} source={require('./assets/CICD.jpg')}></Image>
        </View>
        <View style={styles.cardContainer}>
          <Text style={styles.headerText}>Vamos a demostrar un flujo de CI/CD con Circle-CI:</Text>
          
        </View>
      <StatusBar style="auto" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
  headerContainer:{
    flex:0.5,
    justifyContent:'center',
    paddingBottom: 80
  },
  cardContainer:{
    backgroundColor: '#f5f5f5',
    width: '90%',
    borderRadius: 10,
    marginBottom: 60,
    shadowColor: "#000",
    shadowOffset: {
      width: 0,
      height: 7,
    },
    shadowOpacity: 0.41,
    shadowRadius: 9.11,
    
    elevation: 14,
    flex:0.3
  },
  headerText:{
    margin:10,
    fontSize: 30,
    color: '#303030',
    fontWeight: 'bold'
  },
  button: {
    backgroundColor: 'white',
    borderColor: '#bdbdbd',
    borderWidth: 2,
    marginLeft: 30,
    marginRight: 30,
    marginTop: 30,
    height: 40,
    borderRadius: 20,
    alignItems: "center",
    justifyContent: 'center',
    shadowColor: 'black',
    shadowColor: "#000",
    shadowOffset: {
        width: 0,
        height: 6,
    },
    shadowOpacity: 0.37,
    shadowRadius: 3.49,
  },
  buttonText:{

  },
  image: {
    marginTop: 100,
    height:300,
    width:400
  },
  
});
