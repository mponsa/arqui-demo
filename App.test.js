import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';


test('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});

describe('<App />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree.children.length).toBe(2);
  });
});

