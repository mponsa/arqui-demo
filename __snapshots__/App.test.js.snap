// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "flex": 0.5,
        "justifyContent": "center",
        "paddingBottom": 80,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 300,
          "marginTop": 100,
          "width": 400,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#f5f5f5",
        "borderRadius": 10,
        "elevation": 14,
        "flex": 0.3,
        "marginBottom": 60,
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 7,
          "width": 0,
        },
        "shadowOpacity": 0.41,
        "shadowRadius": 9.11,
        "width": "90%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#303030",
          "fontSize": 30,
          "fontWeight": "bold",
          "margin": 10,
        }
      }
    >
      Vamos a demostrar un flujo de CI/CD con Circle-CI:
    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderColor": "#bdbdbd",
          "borderRadius": 20,
          "borderWidth": 2,
          "height": 40,
          "justifyContent": "center",
          "marginLeft": 30,
          "marginRight": 30,
          "marginTop": 30,
          "opacity": 1,
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 6,
            "width": 0,
          },
          "shadowOpacity": 0.37,
          "shadowRadius": 3.49,
        }
      }
    >
      <Text
        style={Object {}}
      >
        Press me!
      </Text>
    </View>
  </View>
</View>
`;
